config/seata-config-core/src/main/java/io/seata/config/AbstractConfiguration.java

```diff
@@ -61,6 +66,22 @@ public long getLong(String dataId) {
         return getLong(dataId, 0L);
     }
 
+    @Override
+    public Duration getDuration(String dataId) {
+        return getDuration(dataId, Duration.ZERO);
+    }
+
+    @Override
+    public Duration getDuration(String dataId, Duration defaultValue) {
+        return getDuration(dataId, defaultValue, DEFAULT_CONFIG_TIMEOUT);
+    }
+
+    @Override
+    public Duration getDuration(String dataId, Duration defaultValue, long timeoutMills) {
```

Comment:
if value=-1 parse error？

Reply:
I have added no unit processing, now it works fine.